pkgs/by-name/lt/lttng-ust/package.nix

```diff
@@ -65,5 +65,4 @@ stdenv.mkDerivation rec {
     platforms = lib.intersectLists platforms.linux liburcu.meta.platforms;
     maintainers = [ maintainers.bjornfor ];
   };
-
```

Comment:
Noise.

Reply:
Can we *NOT* do whitespace nits??? Please?

Reply:
Because I had to split up the file after the fact, the space change belonged in the second commit.